Cargo.lock

```diff
@@ -21,24 +38,91 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
+name = "clap"
+version = "3.1.18"
```

Comment:
Clap had major updates since you had been adding these dependencies.  When installing the application as described in its README, the compiler will yell at you since your code uses deprecated features, at the moment.

Run `cargo update` and fix arising issues like I did in https://github.com/kevinmatthes/cargo-optimise/commit/0f939ae6bc9d8d22a389936cbfd0cbddc4daca44:

* `bool`s need to be annotated `action`, now.
* If some options need to be parsed, they need to be annotated `value_parser`, now.

I recommend you to do this update since you will not only see the compiler warnings when installing but are also introducing a completely new dependency with this Pull Request.  So in my opinion, when merged, the Pull Request should only rely on state-of-the-art dependencies, this is, in our case, the latest available `clap`.

Reply:
For further information on how to fix arising issues from the latest Clap, please consult the Clap tutorials in the official GitHub repository; this is also where I found the solution I implemented in the named commit.

Reply:
I updated to version `3.2.3` and it compiles and runs just fine. And since I took this code straight from their tutorial, I think it is still alright.

Reply:
If `cargo b` does not give you any warnings, everything will be indeed alright.